### DISCUSSION TITLE
Repeat Advanced feature request

### DISCUSSION THREAD

**Discussion by mi-hla** [ProxymanApp/Proxyman#2148] [Ideas]
Hi, 
I would like to add to the Proxyman **Repeat Advanced** feature similar to the one from Charles Proxy. It is good for quick API checks (for example stability or duration without the copy cURL and using another tool).

Michal

<img width="652" alt="Snímek obrazovky 2024-10-04 v 16 36 53" src="https://github.com/user-attachments/assets/151dbab0-047a-4355-ae8b-7caa5c7dde6c">

**Comment by NghiaTranUIT** [comment]
Thanks, let's keep this ticket open. If we have more request on this feature, we will support it 👍

**Comment by dannyyy** [comment]
I eagerly was looking for such a feature, to check behavior in case of real double postings.
